compiler/test/run.d

```diff
@@ -427,7 +427,9 @@ Target[] predefinedTargets(string[] targets)
                 break;
 
             case "all":
-                newTargets ~= createUnitTestTarget();
+                version (FreeBSD) { /* ??? unittest runner fails for no good reason on GHA. */ }
+                else
+                    newTargets ~= createUnitTestTarget();
```

Comment:
I say this is for no good reason - however when commenting out [deinitialization.d:152](https://github.com/dlang/dmd/blob/08821bb7e55c3d7a183f8f02c95f02340693ba58/compiler/test/unit/deinitialization.d#L152) explicitly the same module then just failed at [deinitialization.d:131](https://github.com/dlang/dmd/blob/08821bb7e55c3d7a183f8f02c95f02340693ba58/compiler/test/unit/deinitialization.d#L131).

This could suggest that al globals have an initial value _other_ than `null`.  Which begs the question: _Why?_, but actually also _Why not?_ - afterall as a security feature, you can tell a compiler to default initialize all variables with a bit mask value (as opposed to zeros or garbage). :shrug:

Reply:
> I say this is for no good reason - however when commenting out [deinitialization.d:152](https://github.com/dlang/dmd/blob/08821bb7e55c3d7a183f8f02c95f02340693ba58/compiler/test/unit/deinitialization.d#L152) explicitly the same module then just failed at [deinitialization.d:131](https://github.com/dlang/dmd/blob/08821bb7e55c3d7a183f8f02c95f02340693ba58/compiler/test/unit/deinitialization.d#L131).
> 
> This could suggest that al globals have an initial value _other_ than `null`. Which begs the question: _Why?_, but actually also _Why not?_ - afterall as a security feature, you can tell a compiler to default initialize all variables with a bit mask value (as opposed to zeros or garbage). Though I am struggling to justify this for global data. But equally surely you're testing it wrong if you're checking the value _before_ it's been initialized. 🤷